hoomd/md/methods.py

```diff
@@ -473,34 +519,72 @@ class Langevin(_Method):
-    2. Specify :math:`\alpha` which scales the particle diameter to
-       :math:`\gamma = \alpha d_i`. The units of
-       :math:`\alpha` are mass / distance / time.
+    1. Specify :math:`\alpha` which scales the particle diameter to 
+       :math:`\gamma = \alpha d_i`. The units of :math:`\alpha` are 
+       mass / distance / time.
+    2. After the method object is created, specify the 
+       attribute ``gamma`` and ``gamma_r`` for rotational damping or random 
+       torque to assign them directly, with independent values for each 
+       particle type in the system.
 
-    *kT* can be a variant type, allowing for temperature ramps in simulation
-    runs.
-
-    A :py:class:`hoomd.compute.thermo` is automatically created and associated
-    with *group*.
 
     Warning:
         When restarting a simulation, the energy of the reservoir will be reset
         to zero.
 
+
     Examples::
 
-        all=hoomd.filter.All()
-        langevin = hoomd.md.methods.Langevin(filter=all, kT=0.2, seed=1, alpha=1.0)
-        integrator = hoomd.md.Integrator(dt=0.001, methods=[langevin], forces=[lj])
+        langevin = hoomd.md.methods.Langevin(filter=hoomd.filter.All(), kT=0.2, 
+        seed=1, alpha=1.0)
+        integrator = hoomd.md.Integrator(dt=0.001, methods=[langevin], 
+        forces=[lj])
+
+
+    Examples of using ``gamma`` or ``gamma_r`` on drag coefficient::
+
+        langevin = hoomd.md.methods.Langevin(filter=hoomd.filter.All(), kT=0.2,
+        seed=1)
+        langevin.gamma.default = 2.0
+        langevin.gamma_r.default = [1.0,2.0,3.0]
+
+
+    Attributes:
+        filter (hoomd.filter._ParticleFilter): Subset of particles to
+            apply this method to.
+
+        kT (hoomd.variant.Variant): Temperature of the
+            simulation (in energy units).
+
+        seed (int): Random seed to use for generating
+            :math:`\vec{F}_\mathrm{R}`.
+
+        alpha (float): When set, use :math:`\alpha d_i` for the drag 
+            coefficient where :math:`d_i` is particle diameter. 
+            Defaults to None.
+
+        gamma (TypeParameter[``particle type``, float]): The drag coefficient 
+            can be directly set instead of the ratio of particle diameter 
+            (:math:`\gamma = \alpha d_i`). The type of ``gamma`` parameter is 
+            either positive float or zero.
+
+        gamma_r (TypeParameter[``particle type``, [float,float,float]]): The 
+            rotational drag coefficient can be set. The type of ``gamma_r``
+            parameter is a tuple of three float. The type of each element of 
+            tuple is either positive float or zero.
+
+        tally_reservoir_energy (bool): If true, the energy exchange
+            between the thermal reservoir and the particles is tracked. Total
+            energy conservation can then be monitored by adding 
+            ``langevin_reservoir_energy_groupname`` to the logged quantities. 
+            Defaults to False.
```

Comment:
We aren't using the C++ logging system, so this actually does nothing for now, because we don't expose the quantity to be logged in Python. @joaander would you rather wait and just remove this from the constructor or expose the reservoir energy to be logged?

Reply:
We can remove it from here in this PR. I'll add the loggable quantity back in on another PR and it will get documented there.

Reply:
I removed it.